### ISSUE TITLE
When I run ./minimal, the error message is minimal <tflite model>. Where is the tflite model

### ISSUE THREAD

**Issue by lw3585** [tensorflow/tensorflow#30088]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:armv7l platform
- TensorFlow installed from (source or binary):source
- TensorFlow version:remotes/origin/r1.13
- Python version:N/A
- Installed using virtualenv? pip? conda?:git
- Bazel version (if compiling from source):N/A
- GCC/Compiler version (if compiling from source):arm-linux-gnueabihf-g++
- CUDA/cuDNN version:N/A
- GPU model and memory:N/A

**Describe the problem**
I have compiled tensoflow-lite on ubuntu and successfully generated libtensorflow-lite.a and minimal, how should I test them on the Raspberry Pi? When I run ./minimal, the error message is minimal <tflite model>. Where is the tflite model, please help me, 3Q

**Provide the exact sequence of commands / steps that you executed before running into the problem**
# ./minimal 
minimal <tflite model>

# ./minimal benchmark
benchmark-lib.a  benchmark_model  
root@am57xx-evm:~/app_virtual# ./minimal benchmark_model 
Model provided has model identifier '', should be 'TFL3'

Error at tensorflow/lite/examples/minimal/minimal.cc:49

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
As above...

**Comment by freedomtan** [comment]
any valid TensorFlow Lite model works, e.g., you can get some models [here](https://www.tensorflow.org/lite/guide/hosted_models)

**Comment by lw3585** [comment]
> any valid TensorFlow Lite model works, e.g., you can get some models [here](https://www.tensorflow.org/lite/guide/hosted_models)

Website can't open。，

**Comment by lw3585** [comment]
I downloaded a model (mobilenet_v1_1.0_224.tflite) from [here](https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models), then modified the Makefile to generate the BIN file label_image, and then run the command ./label_image -v 1 -m ./mobilenet_v1_1.0_224.tflite -i ./grace_hopper. bmp -l ./labels.txt, test is ok..

**Comment by gadagashwini** [comment]
Looks like its resolved. Can we close this issue?

**Comment by lw3585** [comment]
> Looks like its resolved. Can we close this issue?
